### PR TITLE
PG allow get_vertex_data to accept single type or id

### LINKED ISSUE #2692: [FEA] Property Graph get_vertex_data accept int
**Describe the solution you'd like and any additional context**
currently get_vertex_data only accepts an array of vertex IDs, it should also support a single int as input

---
_For Developers below this line_

  - [] Code passes CI
  - [] Code uses Graph Primitives
  - [] Code in C++/CUDA layer
  - [] Code in C layer
  - [] Code in pylibcugraph Python layer
  - [] Code in cugraph Python layer
  - [] Documentation
  - [] Test cases at each layer (for MG tests, a note in the PR description indicating the new/existing MG tests were run and passed)

### PR DESCRIPTION
**Pull request by eriknw**
Closes #2692.

Currently SG only.  ~MG version coming shortly!~  MG version done.

### CODE REVIEW

File: python/cugraph/cugraph/dask/structure/mg_property_graph.py
Comment (rlratzel) [inline python/cugraph/cugraph/dask/structure/mg_property_graph.py:449]: is this FIXME still relevant, since it seems like we decided we aren't coercing to a list?  Same question for all the other FIXMEs like this.

**Review by rlratzel — APPROVED** [review]
I had a question about a few old FIXMEs which need not hold up approval.

File: python/cugraph/cugraph/dask/structure/mg_property_graph.py
Comment (eriknw) [reply python/cugraph/cugraph/dask/structure/mg_property_graph.py:449]: well, we check if it's a string, but we don't yet check if it's a list.  We defer to `isin`.  so if we don't have a clear answer to this FIXME, then it _may_ still be relevant.

**Comment by rlratzel** [comment]
@gpucibot merge

**Pull request merged**
